modules/administration-guide/pages/managing-workloads-using-the-che-server-api.adoc

```diff
@@ -1,17 +1,19 @@
 :_content-type: PROCEDURE
-:description: Managing {prod-short} server workloads using the {prod-short} server API
+:description: Managing {prod-short} server and {prod-short} dashboard workloads using the API
 :keywords: administration-guide, api
 :navtitle: Using the {prod-short} server API
 :page-aliases:
 
 [id="managing-workloads-using-the-{prod-id-short}-server-api"]
-= Managing {prod-short} server workloads using the {prod-short} server API
+= Managing {prod-short} server and {prod-short} dashboard workloads using the API
 
-To manage {prod-short} server workloads, use the Swagger web user interface to navigate {prod-short} server API.
+To manage {prod-short} server and {prod-short} dashboard workloads, use the Swagger web user interface to navigate API.
 
 .Procedure
 
-* Navigate to the Swagger API web user interface: `pass:c,a,q[{prod-url}]/swagger`.
+* Navigate to the Swagger API web user interface:
+ - `pass:c,a,q[{prod-url}]/swagger`   ({prod-short} server)
+ - `pass:c,a,q[{prod-url}]/dashboard/api/swagger`   ({prod-short} dashboard)
 
```

Comment:
@olexii4 @deerskindoll I believe we need an `IMPORTANT` section here that e.g.

IMPORTANT:  DevWorkspace is a k8s object and manipulations should happen on the Kubernetes API level - https://eclipse.dev/che/docs/stable/end-user-guide/managing-workspaces-with-apis/